app/views/thredded/shared/_page.html.erb

```diff
@@ -1,6 +1,7 @@
 <div id="thredded--container" class="thredded--main-container <%= yield(:thredded_page_id) %><%= ' thredded--is-moderator' if moderatable_messageboards_ids.present? %>" data-thredded-page-id="<%= yield(:thredded_page_id) %>">
   <%= render 'thredded/shared/header' %>
   <%= render 'thredded/shared/flash_messages' %>
+  <div id="thredded--definitions" data-thredded-root="<%= thredded.root_path%>"></div>
```

Comment:
Should this perhaps be `data-thredded-root-url` and defined on the `#thredded--container` div, as we already have some data attributes for JS defined there?